tests/units/test_apis_plugins.py

```diff
@@ -734,3 +734,59 @@ def run():
                 )
 
         run()
+
+
+class TestApisPluginDedtLumifApi(BaseApisPluginTest):
```

Comment:
`dedt_lumi` does not use an Api plugin, but `ECMWFSearch` search plugin. Move this test to `tests/units/test_search_plugins.py`